cmd/oceanbench/broadcast.go

```diff
@@ -151,6 +151,7 @@ type BroadcastConfig struct {
 	Transitioning     bool          // If the broadcast is transition from live to slate, or vice versa.
 	StateData         []byte        // States will be marshalled and their data stored here.
 	Account           string        // The YouTube account email that this broadcast is associated with.
+	InFailure         bool          // True if the broadcast is in a failure state.
```

Comment:
`InFailure` means we are in a serious failure state, not just some fleeting failure, right?

If so, I think the term `CriticalFailure` might better the severity of the situation.

Reply:
We kind of have unhealthy -> infailure -> broadcast disabled. Broadcast disabled is probably more akin to a critical failure because it's at the point where we can't do anything. This normally happens with direct broadcasts because we don't have the option of a slate. I'm personally not super settled on this though, and things will likely get clearer when we add better and more error condition handling; can discuss further.